package.json

```diff
@@ -0,0 +1,67 @@
+{
+	"name": "@ckeditor/ckeditor4-vue",
+	"version": "0.0.0",
+	"main": "dist/ckeditor.js",
+	"devDependencies": {
+		"@babel/core": "^7.6.4",
+		"@babel/preset-env": "^7.6.3",
+		"@vue/test-utils": "^1.0.0-beta.29",
+		"babel-loader": "^8.0.6",
+		"chai": "^4.1.2",
+		"core-js": "^3.4.5",
+		"eslint": "^5.5.0",
+		"eslint-config-ckeditor5": "^1.0.8",
+		"istanbul-instrumenter-loader": "^3.0.0",
+		"karma": "^4.4.1",
+		"karma-browserstack-launcher": "^1.3.0",
+		"karma-chai": "^0.1.0",
+		"karma-chrome-launcher": "^2.2.0",
+		"karma-coverage": "^1.1.1",
+		"karma-coveralls": "^2.0.0",
+		"karma-firefox-launcher": "^1.2.0",
+		"karma-mocha": "^1.3.0",
+		"karma-mocha-reporter": "^2.2.4",
+		"karma-sinon": "^1.0.5",
+		"karma-sourcemap-loader": "^0.3.7",
+		"karma-webpack": "^3.0.5",
+		"live-server": "^1.2.1",
+		"minimist": "^1.2.0",
+		"mocha": "^5.2.0",
+		"sinon": "^7.5.0",
+		"terser-webpack-plugin": "^1.1.0",
+		"vue": "^2.5.17",
+		"vue-router": "^3.1.3",
+		"vue-template-compiler": "^2.5.17",
+		"webpack": "^4.41.2",
+		"webpack-cli": "^3.3.9"
+	},
+	"peerDependencies": {
+		"vue": "^2.5.17"
+	},
+	"dependencies": {
+		"load-script": "^1.0.0"
+	},
+	"engines": {
+		"node": ">=8.0.0",
+		"npm": ">=5.7.1"
+	},
+	"scripts": {
+		"start": "npm run build && live-server --open=samples/index.html",
+		"build": "webpack --mode production",
+		"develop": "webpack --mode development --watch",
+		"test": "karma start",
+		"preversion": "npm test",
+		"version": "npm run build && git add -f dist/",
```

Comment:
Running `npm version` also runs `preversion` and `postversion` scrips: https://github.com/ckeditor/ckeditor4-vue/blob/165df1607e53f3cdbf4741704a537ca6e9a43213/package.json#L53-L55

`npm version` also takes care of creating version tag and committing it to a repository, see https://docs.npmjs.com/cli/version#description

Reply:
Ok, I thought that we somehow redefined `version` so it will only run what is declared here. The docs says it will run default `npm version` and then our custom part so 👍 

We should utilize it in other integrations too ⚙️